### DISCUSSION TITLE
CDN version vs Standalone CLI Ruby on Rails

### DISCUSSION THREAD

**Discussion by fernandokbs** [tailwindlabs/tailwindcss#10000] [Help] [answered]
I have a weird problem using tailwind for rails, it has a weird behavior some classes dont work, but if instead I add the CDN in the main layout it has anothe behavior wich is the correct one, I am using 3.2.4 latest version.

I am reading that Standanlone CLI is a solution for the no nodejs env for rails and phoenix, maybe an error for this?

First image contains Standanlone CLI version which says is 3.2.4
 
![alt text](https://i.imgur.com/buLxFJT.png)

This second image overrides the default one with the CDN version 3.2.4

![alt text](https://i.imgur.com/ljEZX2k.png)

Any ideas for this?

**Comment by wongjn** [comment, marked as answer]
Seems like either the `content` file globs in the Tailwind configuration do not sufficiently cover your code that has Tailwind classes or the other CSS files that load after the `tailwind-*.css` file override its declarations.
